code/__defines/mobs.dm

```diff
@@ -123,16 +123,16 @@
 #define MOB_MINISCULE	1
 
 #define BASE_MAX_NUTRITION	400
-#define HUNGER_FACTOR		0.025 // Factor of how fast mob nutrition decreases. Moved here from chemistry define
+#define HUNGER_FACTOR		0.04 // Factor of how fast mob nutrition decreases over time.
```

Comment:
You are aware that this almost doubles the hunger factor ?

Reply:
Hunger factor before the Thirst PR was 0.05. This was originally a change to see if the bug was caused by a reduced hunger factor but I decided to keep it in as it reflected the intended change of what hunger and thirst was meant to be.